testing/tests/ui/caller_arguments.stderr

```diff
@@ -0,0 +1,59 @@
+error: missing `c` argument
+ --> InvalidNumberArguments.txt:3:18
+       "(\"a\", \"b\") -}}\n    {%- endmacro -%}\n    {%- call(a,b,c) test() -%}\n        {{- a"...
+  --> tests/ui/caller_arguments.rs:5:14
+   |
+5  |       source = r#"
+   |  ______________^
+6  | |     {% macro test() %}
+7  | |         {{- caller("a", "b") -}}
+8  | |     {%- endmacro -%}
+...  |
+11 | |     {%- endcall -%}
+12 | |     "#,
+   | |______^
```

Comment:
this does not look quite correct it should be the call

Reply:
Strange indeed, seems like the span is created from the start of the arguments and not of `caller`.